app/presenters/reservation_user_action_presenter.rb

```diff
@@ -23,7 +23,7 @@ def user_actions
 
     if can_switch_instrument?
```

Comment:
Since `Reservation#can_switch_instrument_on?` doesn't consider `current_user`, it could be true for operators. Same goes for `accessories?`. Stepping back, it's looking more like all the sharing between the user and admin interfaces made things more complicated.